AngularNoNgAttributesSnapshotSerializer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
+
+const attributesToRemovePatterns = ['ng-reflect', '_nghost', '_ngcontent', 'ng-version'];
+
+const serialize = (node, ...rest) => {
+  const nodeCopy = node.cloneNode(true);
+  Object.values(nodeCopy.attributes)
+    .map(attribute => attribute.name)
+    .filter(attributeName =>
+      attributesToRemovePatterns
+        .some(removeAttributePattern => attributeName.startsWith(removeAttributePattern))
+    )
+    .forEach(attributeNameToRemove => nodeCopy.attributes.removeNamedItem(attributeNameToRemove));
+
+  return jestDOMElementSerializer.serialize(nodeCopy, ...rest);
+};
+
+const serializeTestFn = (val) => {
+  return val.attributes !== undefined && Object.values(val.attributes)
+    .map(attribute => attribute.name)
```

Comment:
I am not sure this will create some bottleneck about perfomance. Like we had `map` + `filter` and it is like `O(2n)`. One `filter` would be `O(n)`. May be I am not right but it is almost same for CPU. It not becomes `O(n^2)` which is really worse. But seems to me for people code is more readable using `map` + `filter`.

Done